sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -973,20 +919,23 @@ public virtual async Task<Response<ConfigurationSetting>> ClearReadOnlyAsync(str
         }
 
         /// <summary>
-        /// Sets an existing <see cref="ConfigurationSetting"/> as read write in the configuration store.
+        /// Specifies whether an existing <see cref="ConfigurationSetting"/> is read only in the configuration store.
```

Comment:
I think it's important to say "Set" here to indicate that state will change as a result of making this call.

What about 
"Sets an existing `ConfigurationSetting` to read only in the configuration store." ?

Would you mind updating the instances of this comment on other methods as well?

Reply:
> "Sets an existing `ConfigurationSetting` to read only in the configuration store."

It wouldn't be clear from summary how to set it to read-write. How about "Sets an existing `ConfigurationSetting` to read only or read write state in the configuration store." ?

Reply:
Ah, yes.  I like that.